### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by johscheuer**
# Description

Updated a few sections in our documentation to improve the wording and the expectations.

## Type of change

*Please select one of the options below.*

- Documentation

## Discussion

-

## Testing

Not needed.

## Documentation

Updated.

## Follow-up

-

### CODE REVIEW

File: docs/manual/getting_started.md [hunk 1]
```diff
@@ -74,18 +103,21 @@ spec:
             - name: FDB_CLUSTER_FILE
               value: /mnt/config-volume/cluster-file
             volumeMounts:
-            - name: config-volume
+            - name: shared-volume
               mountPath: /mnt/config-volume
```
Comment (nicmorales9) [inline docs/manual/getting_started.md:107]: doesn't match the name above?

**Review by nicmorales9 — APPROVED** [review]
check the one question, but the changes look good to me! thanks

File: docs/manual/getting_started.md (hunk 1 above)
Comment (johscheuer) [reply docs/manual/getting_started.md:107]: The init container mounts the `config-volume` which is a `ConfigMap` and copies the cluster file into the `shared-volume` which is an `EmptyDir`. The reason for this is to make the cluster file writable by the client.

Comment (nicmorales9) [reply docs/manual/getting_started.md:107]: yep that makes sense. It might be helpful to have a comment for those of us who always mix these things up

Comment (johscheuer) [reply docs/manual/getting_started.md:107]: I'll add a comment in another PR, I still have to create a note on why the cluster file should always be copied and not directly be used by the application: https://github.com/FoundationDB/fdb-kubernetes-operator/issues/1958#issuecomment-1985554885.

Comment (nicmorales9) [reply docs/manual/getting_started.md:107]: even better!

**Pull request merged**